### PR TITLE
Fix the capturing tests to match the latest spec

### LINKED ISSUE #148: Update tests for reduce-hit-tests changes
We merged #129 and #138 to gh-pages, but still need to update the web-platform-tests to handle implicit capture and the lack of boundary events during capture.

### PR DESCRIPTION
**Pull request by NavidZ**
closes w3c/pointerevents#148

### CODE REVIEW

**Comment by NavidZ** [comment]
@mustaqahmed

File: pointerevents/pointerevent_capture_suppressing_mouse-manual.html [hunk 1]
```diff
@@ -18,10 +18,11 @@
...
                 <li> Move your mouse out of the purple rectangle. pointerout and pointerleave should be logged inside of it</li>
-                <li> Press and hold left mouse button over "Set Capture" button. "gotpointercapture" should be logged in the black rectangle</li>
+                <li> Press and hold left mouse button over "Set Capture" button and move mouse a litte inside the button. "gotpointercapture", "pointerover", and "pointerenter" should be logged in the black rectangle</li>
                 <li> Put your mouse over the purple rectangle and then move it out. Nothing should happen</li>
-                <li> Put your mouse over the black rectangle. pointerover and pointerenter should be logged inside of it.</li>
-                <li> Move your mouse out of the black rectangle. pointerout and pointerleave should be logged inside of it</li>
+                <li> Put your mouse over the black rectangle and then move it out. Nothing should happen.</li>
+                <li> Put your mouse over the purple rectangle and then release left mouse button. "lostpointercapture" should be logged in the black rectangle and pointerover</li>
```
Comment (mustaqahmed) [inline pointerevents/pointerevent_capture_suppressing_mouse-manual.html]: The last two words look like a leftover: "and pointerover".

File: pointerevents/pointerevent_capture_suppressing_mouse-manual.html [hunk 2]
```diff
@@ -18,10 +18,11 @@
...
+                <li> Move your mouse in the purple rectangle a little. "pointerout" and "pointerleave" should be logged in the black rectangle. Also "pointerover" and "pointerenter" should be logged in the purple rectangle"</li>
```
Comment (mustaqahmed) [inline pointerevents/pointerevent_capture_suppressing_mouse-manual.html]: Combining this step with above makes it easier. The event description would become too long but it doesn't matter, right?

File: pointerevents/pointerevent_capture_suppressing_mouse-manual.html [hunk 3]
```diff
@@ -18,10 +18,11 @@
...
                 <li> Release left mouse button. "lostpointercapture" should be logged in the black rectangle</li>
```
Comment (mustaqahmed) [inline pointerevents/pointerevent_capture_suppressing_mouse-manual.html]: This line is redundant now.

**Review by mustaqahmed — COMMENTED** [review]
Please update the commit description to add: "As per the current spec, boundary events are not fired during a capture."

File: pointerevents/pointerevent_capture_suppressing_mouse-manual.html [hunk 4]
```diff
@@ -89,15 +90,15 @@
                         test_pointerover_capture.done();
                         if (!isRelatedTargetValueTested) {
                             test(function() {
-                                assert_true(event.relatedTarget==null, "relatedTarget is null when the capture is set")
-                            }, "relatedTarget is null when the capture is set. relatedTarget is " + event.relatedTarget);
+                                assert_true(event.relatedTarget != null, "relatedTarget should behave the same as when the capture is not set")
```
Comment (RByers) [inline pointerevents/pointerevent_capture_suppressing_mouse-manual.html]: nit: you get better errors if you [use assert_not_equals](https://github.com/w3c/testharness.js/blob/master/docs/api.md#list-of-assertions)

File: pointerevents/pointerevent_capture_suppressing_mouse-manual.html [hunk 5]
```diff
@@ -89,15 +90,15 @@
...
+                            }, "relatedTarget is not null for boundary events even when the capture is set.");
                             isRelatedTargetValueTested = true;
                         }
                         var hitTest = document.elementFromPoint(event.clientX, event.clientY);
                         if(event.target !== hitTest && !isTargetAuthenticityTested) {
                             test(function () {
-                                assert_unreached("pointerover for this target shouldn't trigger events on capture target");
-                            }, "pointerover should only trigger over the black rectangle");
+                                assert_true(event.target !== hitTest, "pointerover should be fired on capture target even if the pointer it not over the capture target");
```
Comment (RByers) [inline pointerevents/pointerevent_capture_suppressing_mouse-manual.html]: assert_not_equals

File: pointerevents/pointerevent_setpointercapture_relatedtarget-manual.html [hunk 6]
```diff
@@ -65,15 +65,14 @@
...
-                // TA: 13.3
+                // After invoking the setPointerCapture method on an element, subsequent pointer events for the specified pointer must be targeted at that element
+                // and boundary events should be sent accordingly and relatedTarget should behave normally.
                 on_event(target0, "pointerover", function (event) {
                     log("pointerover", document.getElementById('target0'));
                     if(isPointerCapture && isPointeroverGot) {
                         test(function() {
-                            assert_true(event.relatedTarget==null, "relatedTarget is null when the capture is set")
-                        }, "relatedTarget is null when the capture is set. relatedTarget is " + event.relatedTarget);
+                            assert_true(event.relatedTarget!=null, "relatedTarget should not be null even when the capture is set")
```
Comment (RByers) [inline pointerevents/pointerevent_setpointercapture_relatedtarget-manual.html]: assert_not_equals

**Review by RByers — COMMENTED** [review]
Looks reasonable (modulo Mustaq's suggestions too).  Thanks!

**Comment by RByers** [comment]
LGTM
